static/css/cf-enhancements.less

```diff
@@ -508,4 +504,152 @@ tbody {
 .expandable_header-right {
   width: 22%;
   text-align: right;
-}
\ No newline at end of file
+}
+
+
+/* topdoc
+  name: Icon list modifier
+  family: cf-typography
```

Comment:
Just putting a comment here that the family is `cf-typography` with the assumption that this will migrate over and go through vetting in https://github.com/cfpb/cf-typography/issues/29